cmd/security-bootstrapper/entrypoint-scripts/kong_wait_install.sh

```diff
@@ -79,5 +79,9 @@ else
   echo "$(date) failed to kong migrations, returned code = " $code
 fi
 
+echo "$(date) Configuring Kong Admin API..."
+# kong config db_import /usr/local/kong/kong.yml
+/docker-entrypoint.sh kong config db_import /usr/local/kong/kong.yml || true
```

Comment:
What's the purpose of `|| true`

Reply:
this is inherited from Beau's PR and part of his PR:  see comment here: https://github.com/edgexfoundry/edgex-go/pull/3328#discussion_r605183806